visavis_vision/WallInfo[] walls